### PR TITLE
miniupnp: update to version 2.2.3

### PR DESCRIPTION
**Pull request by shalzz**
With refreshed patches

### CODE REVIEW

**Comment by shalzz** [comment]
This version fixes the MS IGDv1 issues (miniupnp/miniupnp#539) and so the IGDv1 variant should probably be removed.

File: net/miniupnpd/Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=miniupnpd
-PKG_VERSION:=2.2.2
+PKG_VERSION:=2.2.3
 PKG_RELEASE:=3
```
Comment (PolynomialDivision) [inline net/miniupnpd/Makefile:12]: reset PKG_RELEASE to 1 again.

**Comment by shalzz** [comment]
Seems like just a patch refresh is not sufficient for building the upnpc package, unfortunately I'm not too familiar with the make/cmake build system to fix this build issue.

**Review by openips — APPROVED** [review]

**Comment by PolynomialDivision** [comment]
Can you seperate that into two commits and prefix it with the correct package name, so `miniupnpc`. If actually the package miniupnp, we should maybe think about combining both packages.

**Comment by shalzz** [comment]
> Can you seperate that into two commits and prefix it with the correct package name, so `miniupnpc`. If actually the package miniupnp, we should maybe think about combining both packages.

Done. Do you want me to create them as separate PRs?

**Comment by PolynomialDivision** [comment]
> Done. Do you want me to create them as separate PRs?

No, everything fine. Even better since they originate from the same project. :) I will wait for CI to pass and then I will merge.

**Comment by stintel** [comment]
Next time please search open PRs first: #17094

**Comment by PolynomialDivision** [comment]
> Next time please search open PRs first: #17094

Sorry. Can you maybe add yourself as PKG_MAINTAINER?

**Comment by stintel** [comment]
> Can you maybe add yourself as PKG_MAINTAINER?

Considering it!

**Comment by stintel** [comment]
After looking at the existing issues for miniupnpd I think I'll pass.

**Pull request merged**